test/helper/helper.go

```diff
@@ -28,7 +28,7 @@ func TestClient() *metalv1.APIClient {
 	configuration.AddDefaultHeader("X-Auth-Token", os.Getenv("METAL_AUTH_TOKEN"))
 	configuration.UserAgent = fmt.Sprintf("metal-cli/test-helper %s", configuration.UserAgent)
 	// For debug purpose
-	// configuration.Debug = true
+	configuration.Debug = true
```

Comment:
I know you're in the process of updating the tests; just remember to turn `Debug` off before this gets merged.